src/rwe/GameScene.h

```diff
@@ -231,9 +231,44 @@ namespace rwe
         bool up{false};
         bool down{false};
 
+        bool leftCtrlDown{false};
+        bool rightCtrlDown{false};
         bool leftShiftDown{false};
         bool rightShiftDown{false};
 
+        bool trackingOn{false};
+        UnitId trackedUnitId;
```

Comment:
I've been thinking about this bit. I think I'd rather combine these together into a single variable `std::optional<UnitId> trackedUnitId` as otherwise `trackedUnitId` isn't well defined when `trackingOn` is false. This isn't a blocker to merging but RWE tries to avoid these types of ill-defined states as much as possible.

Reply:
Ah yes, perfect use for optional. Let me add it.

Reply:
Actually, I take it back - in OTA, after a track is stopped (e.g. from edge scroll), pressing T again will return to tracking mode on the _next_ unit in the selection; i.e. the game remembers the last unit that was tracked, so we do need to hold onto trackedUnitId even when trackingOn == false